src/components/Movie/Movie.js

```diff
@@ -1,13 +1,16 @@
 import React from "react"
 import './Movie.css'
+import { Link } from 'react-router-dom'
 
 const Movie = (props) => {
     const { id, image } = props
     return(
+        <Link to={`/${id}`}>
```

Comment:
This is a great way to incorporate Link - it encapsulates the whole poster, rather than just the button!